source/get-started/primer.txt

```diff
@@ -0,0 +1,116 @@
+.. _scala-primer:
+
+======
+Primer
+======
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: code example, design, types, preparation
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+This guide provides background about the {+driver-short+} and its
+asynchronous API before showing you how to use the driver and MongoDB in
+the 
+
+.. :ref:`Quick Start guide <>`.
```

Comment:
Just verifying that this ref is commented out (leaving an unfinished sentence) on purpose?